src/80211_if.c

```diff
@@ -2941,6 +2974,13 @@ static int uccp420wlan_init(void)
 		}
 	}
 
+	error=start_core(hw);
```

Comment:
Yes it will not work, as the current FW by defaults disables coex unless driver sets it.
We will change this FW behavior in the next release and then you can test it again.